.cirrus.yml

```diff
@@ -1,133 +1,186 @@
 ---
 
 env:
-    DEST_BRANCH: "main"
-    GOPATH: "/var/tmp/go"
-    GOBIN: "${GOPATH}/bin"
-    GOCACHE: "${GOPATH}/cache"
-    GOSRC: "${GOPATH}/src/github.com/containers/podman"
-    CIRRUS_WORKING_DIR: "${GOPATH}/src/github.com/containers/podman-py"
-    SCRIPT_BASE: "./contrib/cirrus"
-    CIRRUS_SHELL: "/bin/bash"
-    HOME: "/root"  # not set by default
-
-    ####
-    #### Cache-image names to test with (double-quotes around names are critical)
-    ####
-    FEDORA_NAME: "fedora-34"
-    PRIOR_FEDORA_NAME: "fedora-33"
-    UBUNTU_NAME: "ubuntu-2104"
-
-    # Google-cloud VM Images
-    IMAGE_SUFFIX: "c6431352024203264"
-    FEDORA_CACHE_IMAGE_NAME: "fedora-${IMAGE_SUFFIX}"
-    PRIOR_FEDORA_CACHE_IMAGE_NAME: "prior-fedora-${IMAGE_SUFFIX}"
-    UBUNTU_CACHE_IMAGE_NAME: "ubuntu-${IMAGE_SUFFIX}"
-
-    # Container FQIN's
-    FEDORA_CONTAINER_FQIN: "quay.io/libpod/fedora_podman:${IMAGE_SUFFIX}"
-    PRIOR_FEDORA_CONTAINER_FQIN: "quay.io/libpod/prior-fedora_podman:${IMAGE_SUFFIX}"
-    UBUNTU_CONTAINER_FQIN: "quay.io/libpod/ubuntu_podman:${IMAGE_SUFFIX}"
```

Comment:
I'm not opposed to an indentation change...other than it will make diffing this vs other repos hard.  Since `.cirrus.yml` shares a lot of the same names/conventions, this is a handy thing to maintain.

Reply:
@cevich My yaml editor got excited. I'll look into restoring indentation.